src/modules/java_version_parser.rs

```diff
@@ -0,0 +1,61 @@
+use nom::{
```

Comment:
I think a better location for this might be `src/modules/utils/java_verison_parser.rs`
What do you think?

Reply:
Moved the parser to `modules/utils`.